src/processors/JsonSchemaInputProcessor.ts

```diff
@@ -212,7 +212,8 @@ export class JsonSchemaInputProcessor extends AbstractInputProcessor {
    * @param schema to simplify to common model
    */
   static convertSchemaToCommonModel(schema: Schema | boolean): Record<string, CommonModel> {
-    const commonModels = simplify(schema);
+    const interpreter = new Interpreter();
```

Comment:
I guess interpreter should be constructed before this method execution. Please remember that creating instance by using keyword `new` is not well optimized in javascript and should be avoided at execution time.

Reply:
We could just have one interpreter instance where we could reset the iterated models, but it's an optimization we can make later if it ever becomes a problem.